apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/StandardGrpcRdaSource.java

```diff
@@ -324,7 +324,7 @@ private boolean isUnexpectedDroppedConnectionException(
   private long getStartingSequenceNumber(RdaSink<TMessage, TClaim> sink)
       throws ProcessingException {
     if (startingSequenceNumber.isPresent()) {
-      return startingSequenceNumber.map(x -> Math.max(MIN_SEQUENCE_NUM, x - 1)).get();
+      return startingSequenceNumber.map(x -> x - 1).get();
```

Comment:
Since we removed negative sequence protection here, should we add one in (or do we have one) to the primary production job work flow?

Reply:
I think it's unnecesaary and now that the jobs need to support them it would complicate the code to have special cases.  I could only happen through direct configuration and we might actually find a use case for that some day.  In sandbox for example.